### DISCUSSION TITLE
Can the Wikipedia pipeline be used out of the box?

### DISCUSSION THREAD

**Discussion by ProjectMoon** [open-webui/pipelines#54] [Q&A]
I have installed the Wikipedia pipeline. It starts up (can see it in Docker logs). But ... does it do anything? Or do I need to add something to it to make it fire?

**Comment by c9482** [comment]
Asked the same simple question on Discord but the community is silent🦗

**Comment by kartguru** [comment]
This guide (https://github.com/open-webui/pipelines/discussions/62) specifically references the Wikipedia pipeline so should help

**Reply by c9482** [reply]
This great. I was monitoring the discussions and must have missed it. Thanks!

**Reply by ProjectMoon** [reply]
I see, it shows up as an external model. I never thought to look there. It makes way more sense now. Thank you!

**Comment by justinh-rahb** [comment]
Whatever you put in as input to the "Wikipedia" model is passed to their search, and the first article returned from what I can see. It's just an example, and an early one at that, it may need some updates and improvements.

**Comment by ProjectMoon** [comment]
I see, it shows up as an external model. I never thought to look there. It makes way more sense now. Thank you!
